gpt_engineer/main.py

```diff
@@ -26,6 +26,21 @@ def load_env_if_needed():
     openai.api_key = os.getenv("OPENAI_API_KEY")
 
 
+def load_prompt(dbs: DBs):
+    if dbs.input.get("prompt"):
+        return dbs.input.get("prompt")
+
+    if dbs.workspace.get("prompt"):
+        dbs.input["prompt"] = dbs.workspace.get("prompt")
+        del dbs.workspace["prompt"]
+        return dbs.input.get("prompt")
```

Comment:
This behavior is super confusing to the user no?

Let's revert this before we merge.

Reply:
I see what you are doing here, but I agree with Anton.  I think it would be better to give the user an error so they have to put the prompt in the correct directory.